tests/test_telcameras_v2.py

```diff
@@ -465,3 +466,47 @@ def test_scramble_counts_with_counts_zero(self):
         self.assertIn(count_agg.count_in_scrambled, (0, 1))
         self.assertIn(count_agg.count_out_scrambled, (0, 1))
         self.assertIn(count_agg.count_scrambled, (0, 1))
+
+    def test_scramble_v2_counts_command(self):
+        baker.make(CountAggregate, _fill_optional=['count_in', 'count_out', 'count'], _quantity=1100)
+        for ca in CountAggregate.objects.all():
+            # ModelBaker doesn't take the smallinteger into account, so we adjust for it here
+            if ca.count_in >= 32767 or ca.count_out >= 32767 or ca.count >= 32767:
+                ca.count_in = randint(0, 1000)
+                ca.count_out = randint(0, 1000)
+                ca.count = randint(0, 1000)
+                ca.save()
+
+            self.assertIsNotNone(ca.count_in)
+            self.assertIsNotNone(ca.count_out)
+            self.assertIsNotNone(ca.count)
+            self.assertIsNone(ca.count_in_scrambled)
+            self.assertIsNone(ca.count_out_scrambled)
+            self.assertIsNone(ca.count_scrambled)
+
+        # Do the scrambling
+        call_man_command('scramble_v2_counts')
+
+        differ_count_in = 0
+        differ_count_out = 0
+        differ_count = 0
+        for ca in CountAggregate.objects.all():
+            self.assertIsNotNone(ca.count_in_scrambled)
+            self.assertIn(ca.count_in_scrambled, (ca.count_in-1, ca.count_in, ca.count_in+1))
+            if ca.count_in_scrambled != ca.count_in:
+                differ_count_in += 1
+
+            self.assertIsNotNone(ca.count_out_scrambled)
+            self.assertIn(ca.count_out_scrambled, (ca.count_out-1, ca.count_out, ca.count_out+1))
+            if ca.count_out_scrambled != ca.count_out:
+                differ_count_out += 1
+
+            self.assertIsNotNone(ca.count_scrambled)
+            self.assertIn(ca.count_scrambled, (ca.count-1, ca.count, ca.count+1))
+            if ca.count_scrambled != ca.count:
+                differ_count += 1
+
+        # Make sure that a significant amount of counts_scrambled were actually changed from the original
+        self.assertGreater(differ_count_in, 600)
+        self.assertGreater(differ_count_out, 600)
+        self.assertGreater(differ_count, 600)
```

Comment:
De scrambles draaien op basis van kans. Een dergelijke test is dus niet helemaal robuust. Daarnaast vind ik een loop van 1100 items iets te overdreven. Ik zou dit met enkele filter queries willen afvangen:

```
# check all records have their scrambled counts set
assert not CountAggregate.objects.filter(Q(count_in_scrambled=None) | Q(count_out_scrambled=None) | Q(count_scrambled=None)).exists()

# check that all scrambled counts are within valid range
assert not CountAggregate.objects.filter(Q(count_in_scrambled__gt=F('count_in')+1) | Q(count_in_scrambled__lt=F('count_in')-1))
# zelfde voor twee andere counts
```

Als laatste kan je nog een query doen om te checken hoeveel er anders zijn, en daar een getal aan hangen (zoals 600 wat je nu doet), als dat echt nodig is.